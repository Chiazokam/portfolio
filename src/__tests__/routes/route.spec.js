import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../../routes';

describe('Routes component', () => {
    it('renders successfully', () => {
    const wrapper = shallow(<Routes />);
    expect(wrapper).toMatchSnapshot();
    })
});
