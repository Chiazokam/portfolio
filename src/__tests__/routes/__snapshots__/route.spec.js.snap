// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes component renders successfully 1`] = `ShallowWrapper {}`;
